src/Tempest/Router/src/IsRoute.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Router;
+
+use Tempest\Http\Method;
+use Tempest\Reflection\MethodReflector;
+
+trait IsRoute
+{
+    private Method $method;
```

Comment:
I'll preface this by saying I don't have a solution, but I'm not a huge fan of this approach because in essence, the route attribute corresponds to the method.

Reply:
Ah, shame. Maybe Brendt has an idea. This is how I interpreted his issue https://github.com/tempestphp/tempest-framework/issues/789.

Reply:
No, I think you interpreted right. I'm just thinking out loud. 😛

Reply:
Keep in mind that this is only true for the default route attributes. The goal is that people make their own to model route groups, and I foresee the method not always being part of the attribute.